tool.go

```diff
@@ -266,6 +258,13 @@ func main() {
 		Use:   "run [gofiles...] [arguments...]",
 		Short: "compile and run Go program",
 	}
+	cmdRun.Flags().AddFlag(flagVerbose)
```

Comment:
Ideally, what I want to do is create all the flags, then add them to the commands that need/use them.

However, I don't see an API to create a standalone flag, only to create one within a flag set, then look it up.

The only change that I see being possible is to create all flags within a dummy flag set, then look them up, and add to individual commands.

I don't see how https://godoc.org/github.com/spf13/pflag#FlagSet.AddFlagSet can be useful. It adds an entire flag set, but most commands need a variety of specific flags added.